.github/workflows/license_check.yml

```diff
@@ -19,7 +19,7 @@ name: License Check
 on:
   pull_request:
     branches: [ master ]
-    types: [ opened, synchronize, reopened ]
+    types: [ opened, edited, synchronize, reopened ]
```

Comment:
I don't think `edited` is needed - so, why should the licence check be checked in the code if the PR's title or description is edited?

https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=edited#pull_request